packages/beluga/beluga.0.8.1/opam

```diff
@@ -0,0 +1,8 @@
+opam-version: "1"
+maintainer: "fferre8@cs.cmgill.ca"
+build: [["ocamlbuild" "-r" "-use-ocamlfind" "-j" "4" "src/beluga/main.native"]]
```

Comment:
indeed it would be better to not hard-code the number of jobs in the opam files. Using `jobs` instead of `"4"` will make opam use the number of cores on the user machine.